docs/website/docs/command-reference/delete-component.md

```diff
@@ -100,3 +121,23 @@ If `odo` finds the resources, it will delete them after user confirmation.
 Otherwise, `odo` will exit with a message stating that it could not find the resources on the cluster.
 
 `--namespace` is optional, if not provided, `odo` will use the current active namespace.
+
+#### Filtering resources to delete
+You can specify the type of resources candidate for deletion via the `--running-in` flag.
+Acceptable values are `dev` (for inner-loop resources) or `deploy` (for outer-loop resources).
+
+<details>
+<summary>Example</summary>
+
+```shell
+$ odo delete component --name my-nodejs --running-in deploy
+Searching resources to delete, please wait...
+This will delete "my-nodejs" from the namespace "my-project".
+ •  The following resources will get deleted from cluster:
+ •      - Deployment: my-simple-nodejs-app
+
+? Are you sure you want to delete these resources? Yes
+ ✓  Deleting resources from cluster [22ms]
+The component "my-nodejs" running in the Deploy mode is successfully deleted from namespace "my-project"
+```
```

Comment:
Can you add this to command-reference/docs-mdx/delete/<filename>.mdx so that it can be later used for test automation?